docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,154 @@
+  documentation, and a second change that adds implementations for
   those interfaces.
 
   For features or code contributions that only make sense as one unit
   but that are too large to fit into one reasonable CR, consider a
-  multi-branch CR model: Create a main branch `feature/big-feature`
+  stacked CR model: Create a main branch `feature/big-feature`
   and a number of secondary branches -- e.g.,
   `feature/big-feature-api`, `feature/big-feature-testing`, etc. --
   that each encapsulate a subset of the functionality and that get
   individually code-reviewed against the `feature/big-feature` branch.
   Once all secondary branches are merged into `feature/big-feature`,
-  create a CR for merging the latter into `develop`. The functional
-  equivalent in Gerrit is a series of reviewed commits against a
-  single `feature/big-feature` branch.
+  create a CR for merging the latter into the main branch.
 
 - Only submit **complete**, **self-reviewed** (by diff), and **self-tested** CRs.
-  In order to save reviewers' time, test the submitted changes (i.e., run your code),
+  In order to save reviewers' time, test the submitted changes (i.e., run the test suite),
   make sure they pass all builds as well as all tests and code quality
-  checks -- both locally and on the build servers -- *before assigning
+  checks -- both locally and on the CI servers -- *before assigning
   reviewers*.
 
-- **Refactoring CRs:** Refactoring CRs may *never* alter behavior;
+- **Refactoring CRs:** Refactoring CRs should not alter behavior;
   conversely, a behavior-changing CR should avoid refactoring and code
   formatting changes. There are multiple good reasons for this:
 
-  - Refactoring CRs are normally very large and will consequently be
-    reviewed at a different depth. Unintended behavior changes can leak into
-    the code base without anyone noticing.
+  - Refactoring CRs often touch many lines and files and will consequently be
+    [reviewed with less attention](https://smartbear.com/learn/code-review/best-practices-for-peer-code-review/).
+    Unintended behavior changes can leak into the code base without anyone noticing.
+
+  - Large refactoring CRs break cherry-picking,
+    rebasing, and other source control magic. It is very onerous to undo a
+    behavior change that was introduced as part of a repository-wide
+    refactoring commit.
 
-  - Large refactoring CRs are not compatible with cherry-picking,
-    rebasing, and other source control magic. It is very hard to undo a
-    behavior change that was introduced as part of a refactoring commit.
+  - Expensive human review-time should be spent on the program logic
+    rather than file naming or code formatting.
 
-  - Expensive, human review-time should be spent on the program logic
-    rather than, say, its formatting or the import order. Style issues can be
-    fixed prior to code reviews with tooling like
-    [gradle-baseline](https://github.com/palantir/gradle-baseline).
```

Comment:
There’s a thing missing here on the style front: code style nits ought to be the business of a checkstyle(-like) or auto-formatting tool.

Maybe also something around expected to find ways to make logic/abstractions/functional decomp follow existing patterns in the codebase (where pattern is the classes/interfaces/functions/naming convention).

Reply:
added the tooling thing

Reply:
will add the `logic/abstractions/functional decomp follow existing patterns` in the section below